client/src/main/java/io/pravega/client/segment/impl/EventSegmentReaderImpl.java

```diff
@@ -98,6 +98,7 @@ private ByteBuffer readEvent(long firstByteTimeoutMillis) throws EndOfSegmentExc
         headerReadingBuffer.clear();
         int read = in.read(headerReadingBuffer, firstByteTimeoutMillis);
         if (read == 0) {
+            log.debug("No of bytes read is {}", read);
```

Comment:
This logs a number which is guarenteed to be 0. 
Also any logs should contain enough information to understand the context. In this case it should probably log the segment name.

Reply:
Done